src/Sync/Singular.php

```diff
@@ -234,4 +236,29 @@ public function clearIndexLanguageByMeta( $metaId, $postId ) {
 		$this->clearIndexLanguage( $postId );
 	}
 
+	/**
+	 * @param string   $newStatus
+	 * @param string   $oldStatus
+	 * @param \WP_Post $post
+	 */
+	public function maybeUnsyncByStatus( $newStatus, $oldStatus, $post ) {
+		if ( $oldStatus === $newStatus ) {
+			return;
+		}
+
+		$postIndexable = $this->indexables->get( Constants::INDEXABLE_SLUG_POST );
+		if ( false === $postIndexable ) {
+			return;
+		}
+
+		$indexablePostStati = $postIndexable->get_indexable_post_status();
```

Comment:
```suggestion
		$indexablePostStatus = $postIndexable->get_indexable_post_status();
```

Reply:
there is also `Stati` in the commit so I don't know if it's a typo or intentional :)

everything else is great :+1:

Reply:
I understand stati is the plural form of status (as opposed to statuses which doesn't really exist). I saw it here:

https://developer.wordpress.org/reference/functions/get_post_stati/